### PR TITLE
Implemented Spring Security with JWT

### PR DESCRIPTION
**Pull request by renukaThink**

### CODE REVIEW

File: day-8-jwt/src/main/java/com/example/springjwt/day8jwt/Day8JwtApplication.java [hunk 1]
```diff
@@ -0,0 +1,26 @@
+package com.example.springjwt.day8jwt;
```
Comment (abhaythink) [inline day-8-jwt/src/main/java/com/example/springjwt/day8jwt/Day8JwtApplication.java:1]: Meaningful package names expected

File: day-8-jwt/src/main/java/com/example/springjwt/day8jwt/Day8JwtApplication.java [hunk 2]
```diff
@@ -0,0 +1,26 @@
...
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.boot.CommandLineRunner;
+import org.springframework.boot.SpringApplication;
+import org.springframework.boot.autoconfigure.SpringBootApplication;
+import org.springframework.security.crypto.bcrypt.BCryptPasswordEncoder;
+
+import java.util.Optional;
+
+@SpringBootApplication
+public class Day8JwtApplication  {
```
Comment (abhaythink) [inline day-8-jwt/src/main/java/com/example/springjwt/day8jwt/Day8JwtApplication.java:16]: App Name expected

File: day-8-jwt/src/main/java/com/example/springjwt/day8jwt/controller/AuthenticationController.java [hunk 3]
```diff
@@ -0,0 +1,33 @@
...
+@NoArgsConstructor
+@RestController
+@RequestMapping("/api/v1/auth")
+@AllArgsConstructor
+class AuthenticationController {
+    @Autowired
+    private AuthenticationServiceImpl authenticationService;
+
+
+    @PostMapping("/sigup")
```
Comment (abhaythink) [inline day-8-jwt/src/main/java/com/example/springjwt/day8jwt/controller/AuthenticationController.java:22]: typo

**Review by abhaythink — APPROVED** [review]

File: day-8-jwt/src/main/java/com/example/springjwt/day8jwt/controller/AuthenticationController.java (hunk 3 above)
Comment (renukaThink) [reply day-8-jwt/src/main/java/com/example/springjwt/day8jwt/controller/AuthenticationController.java:22]: i will correct it

File: day-8-jwt/src/main/java/com/example/springjwt/day8jwt/Day8JwtApplication.java (hunk 2 above)
Comment (renukaThink) [reply day-8-jwt/src/main/java/com/example/springjwt/day8jwt/Day8JwtApplication.java:16]: okay sir

File: day-8-jwt/src/main/java/com/example/springjwt/day8jwt/Day8JwtApplication.java (hunk 1 above)
Comment (renukaThink) [reply day-8-jwt/src/main/java/com/example/springjwt/day8jwt/Day8JwtApplication.java:1]: okay sir

**Pull request merged**